backend/src/main/java/com/soen341/instagram/exception/account/AccountExistException.java

```diff
@@ -0,0 +1,13 @@
+package com.soen341.instagram.exception.account;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
+public class AccountExistException extends RuntimeException
```

Comment:
Just a 'nice to have' modification: add an "S" to exists such that it becomes `AccountExistsException` since the error is thrown if an account under a certain authentification already exists and the user still tries to make an account

Reply:
It's small but I agree. if only for aesthetic purposes.

Reply:
I'm going to remove this exception class. The reason is that I have a new method to retrieve the data so it won't throw any exception.